### ISSUE TITLE
Shader not connected to master node

### ISSUE THREAD

**Issue by LauraWebdev** [simplestargame/VRM-Toon-for-Universal-RP#1]
Hey, I'm trying to make this shader work for Unity 2019.3's UWRP. As you can see, none of the nodes are connected to any masternode, infact there is no masternode at all in the shadergraph.

![image](https://user-images.githubusercontent.com/7365286/66262106-4f401f80-e7da-11e9-83e3-db4af7396d12.png)

I tried fixing it with the provided screenshot in the LWRPToonVRM repo, however it is too small to see which nodes are connected to which properties.

**Comment by simplestargame** [comment]
Recently, Universal RP has updated to version 7.1. There were structural updates in an implements.
My shader code was tested until Universal RP 7.0.1 in Unity 2019.3.0b4. For now, my shaders are not working with Unity 2019.3.0.b6.
Now... I read new structure of pipeline implements, but you will wait for a moments.

**Comment by LauraWebdev** [comment]
Thank you for your quick answer! Do you have a screenshot of the original connections, maybe I can fix it on my end for now.

**Comment by simplestargame** [comment]
I have this one. is it ok?
![image](https://user-images.githubusercontent.com/20289559/66413689-d2f44900-ea32-11e9-92c5-d93a60e30949.png)

**Comment by LauraWebdev** [comment]
Yes, thank you very much! I'll provisionally implement it into my shader until you are able to look into it.

**Comment by simplestargame** [comment]
Hi! AndreasWebdev.
Today, I updated master branch for Up to date Universal Render Pipeline v7.1.2.
Check README steps to fix and use my Toon Shader.
